src/Device.js

```diff
@@ -510,10 +511,10 @@ export default class Device extends EventEmitter {
 
   _setMeasurementPeriod(measurementPeriodInMicroseconds) {
     const command = new Uint8Array(commands.SET_MEASUREMENT_PERIOD);
+    const minMeasurementPeriodinMicroseconds = this.minMeasurementPeriod * 1000;
```

Comment:
You should probably camelCase the I for In: `Periodin` should be `PeriodIn`

Reply:
I know I saw that after we posted. I think I will just fix it with the next release